### ISSUE TITLE
config.json: invalid config keys

### ISSUE THREAD

**Issue by alternico** [ansible-collections/ansible-consul#138]
Hi,

I have just download the ansible role and, running it on Ubuntu 16.10 and using the latest Consul version (1.0.2), I got the following two issues:
```
/usr/local/bin/consul agent     -config-file=/etc/consul/config.json     -config-dir=/etc/consul/consul.d     -pid-file=/var/run/consul/consul.pid
==> Error parsing /etc/consul/config.json: 1 error(s) occurred:

* invalid config key consul_version
```

```
/usr/local/bin/consul agent     -config-file=/etc/consul/config.json     -config-dir=/etc/consul/consul.d     -pid-file=/var/run/consul/consul.pid
==> performance.raft_multiplier cannot be 0. Must be between 1 and 10
```

Thanks

**Comment by brianshumate** [comment]
Hey there,

It looks like you're running the role from GitHub master as the `performance` configuration stuff is not in any release on https://galaxy.ansible.com yet. You should prefer running Ansible Galaxy releases rather than straight from GitHub master if you want stability in the role.

I have committed a potential fix to GitHub master if you'd like to pull and try again.

**Comment by brianshumate** [comment]
These are fixed in v1.30.1 of the role and GH master now.

**Comment by brianshumate** [comment]
Going to close this since it has been fixed now.
